openstack/image/v2/images/results.go

```diff
@@ -102,6 +102,7 @@ func (r *Image) UnmarshalJSON(b []byte) error {
 	type tmp Image
 	var s struct {
 		tmp
+		Properties                  string `json:"properties"`
```

Comment:
I think it must cover both cases, when properties is a string or a map.

Reply:
Ok, I see. Just to clarify, which OpenStack component actually consumes these image `properties`? Nova? I've been looking through the Nova source and documentation, and it appears that `hypervisor_type` is expected to be a top-level image attribute, not nested under `properties`.
According to the Nova scheduling documentation, the scheduler looks for `hypervisor_type` directly on the image object: https://github.com/openstack/nova/blob/3e5a2dbad73ff61d27db8750480e2c0b52133676/doc/source/admin/scheduling.rst?plain=1#L504